stripe/src/main/java/com/stripe/android/model/BankStatuses.kt

```diff
@@ -1,11 +1,11 @@
 package com.stripe.android.model
 
-import com.stripe.android.view.FpxBank
+import com.stripe.android.view.Bank
 import kotlinx.parcelize.Parcelize
 import org.jetbrains.annotations.TestOnly
 
 @Parcelize
-internal data class FpxBankStatuses internal constructor(
+internal data class BankStatuses internal constructor(
```

Comment:
Does Netbanking have a concept of online/offline bank status?

Reply:
I was informed that it might later on but for the beta it will just be a hardcoded list